client/src/styles/bundle.scss

```diff
@@ -1,7 +1,6 @@
-@import "~bootstrap/scss/bootstrap";
-
 // Import variables
-@import "variables"; // from silverstripe/admin
+@import "variables"; // from silverstripe/admin - must be before bootstrap
+@import "~bootstrap/scss/bootstrap";
```

Comment:
Some bootstrap styling wasn't obeying our admin variables - updating this order resolved that problem